CONTRIBUTING.md

```diff
@@ -73,6 +73,10 @@ There will be a notification in the slack channel `#dsp-callisto-internal` once
 
 The terra-docker image hash update [job](https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/)  mentioned above determines which images (including derived images) have been changed and builds each image with the appropriate tag.
 
+## Updating terra-docker-versions-*.json Best Practices
+
+As of 6/8/2022, we have added a new process for updating the image versions available across different environments. Once the Leo PR is merged into develop, the user will need to generate the `terra-docker-versions-new.json` with the new images and copy it by doing `gsutil cp xxx/terra-docker-versions-new.json terra-docker-image-documentation/terra-docker-versions-candidate.json`. The remainder of the update process will be covered through Leo deployment. As part of reliability changes, the versions file will live in the corresponding bucket with the following name convention: `terra-docker-image-documentation-{environment}`.
```

Comment:
can you have the full path here instead `xxx`?

Reply:
The path here would be where the user has the -new.json file. I'm not sure where the file would actually get generated in this case. I think the custom image job should generate the file into the terra-docker-image-documentation bucket, but I don't think it's worked for a little. I can update it in any case

Reply:
we should update it to what the CI generates it to...even if ppl are doing it manually when CI doesn't work, we should generate it in the right place instead of random places